examples/values.yaml

```diff
@@ -0,0 +1,65 @@
+---
+global:
+  appsSubdomain: $OPENSHIFT_APPS_SUBDOMAIN
+configs:
+  cosign_deploy:
+    enabled: true
+  fulcio:
+    namespace_create: false
+  rekor:
+    namespace_create: false
+    # (OPTIONAL) when providing contents of secret with configs.rekor.server.signer
+    signer:
+      secret:
+        # must provide 1 of each, key_file path or key contents
+        # name must match rekor signer secretName in scaffold.rekor.server
+        name: ""
+        #private_key_file: "keys-cert/rekor_key.pem"
+        private_key: ""
+
+# github.com/sigstore/helm-charts/charts
+scaffold:
+  trillian:
+# Trillian allows for the capabilities to bring your own database. This is
+# useful for production deployments where you want to use a managed database
+# service. 
```

Comment:
This is very nitpicky, but could this be moved to above ` trillian:` on line 22, and indented to conform to the convention of all of the other comments in the file?

Reply:
patched!